_data/projects/baker.yml

```
@@ -0,0 +1,16 @@
+---
+name: Baker
+desc: Baker is a library that reduces the effort to orchestrate (micro)service-based process flows
+site: https://ing-bank.github.io/baker/
+tags:
+  - java
+  - scala
+  - petri-nets
+  - jvm
+  - micro-service
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ing-bank/baker/issues?q=is%3Aissue+is%3Aopen+label%3AHacktoberfest
```

We can simplify this link and generate the stats below using the GitHub API:

```suggestion
  link: https://github.com/ing-bank/baker/labels/Hacktoberfest
```